src/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "opc-plc": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "--aa --sph --fn 10 "

```

Comment:
Did you intend to check-in this file?

Reply:
actually it is already checked in. i added it to .gitignore in this checkin. let me try to remove i already with this PR from the repo.